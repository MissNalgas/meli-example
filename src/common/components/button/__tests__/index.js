import renderer from 'react-test-renderer';
import Button from '../';

test('renders button correctly', async () => {
	const tree = renderer.create(<Button>button</Button>).toJSON();
	expect(tree).toMatchSnapshot();
});
