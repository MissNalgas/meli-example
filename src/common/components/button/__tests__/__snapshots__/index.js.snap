// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button correctly 1`] = `
<button
  className="rounded bg-secondary text-lg text-white w-full h-12 hover:bg-secondary-200 transition undefined"
>
  button
</button>
`;
